source/MRViewer/MRFileDialog.cpp

```diff
@@ -560,7 +561,7 @@ void saveFileDialogAsync( std::function<void( const std::filesystem::path& )> ca
     std::string accumFilter = webAccumFilter( params.filters );
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdollar-in-identifier-extension"
-    EM_ASM( download_file_dialog_popup( UTF8ToString( $0 ), UTF8ToString( $1 ) ), params.fileName.c_str(), accumFilter.c_str() );
+    EM_ASM( download_file_dialog_popup( UTF8ToString( $0 ), UTF8ToString( $1 ), $2 ), params.fileName.c_str(), accumFilter.c_str(), MR::ColorTheme::getPreset() == MR::ColorTheme::Preset::Light );
```

Comment:
mb its better to create js function to request theme from cpp, and call it inside js, instead of passing argument, mb in separate taks